### ISSUE TITLE
Feature Request: Profiles

### ISSUE THREAD

**Issue by jonasnick** [Kixunil/btc-rpc-proxy#3]
Would be useful to have a collection of profiles available somewhere, so users don't need to figure out the exact calls that need to be whitelisted for c-lightning, lnd, liquid, etc...

**Comment by Kixunil** [comment]
Would it be enough to have the config files published? (Like what I did with public.)
Note that btc-rpc-proxy supports `--conf-dir` parameter which allows you to load each user from a different file. That way, it's just matter of copying and setting the password.

**Comment by jonasnick** [comment]
Sure, that would work. Doesn't have to be in this repo, but it would make btc-rpc-proxy much easier to use.

**Comment by Kixunil** [comment]
Yeah, I agree. I'm thinking about making a new directory and filling it with examples.

To you have anything interesting already? I've only used the public one so far.

**Comment by jonasnick** [comment]
Ha, I realized just now that your example exactly includes the public RPCs. I guess that's good enough ~~for me~~ for now. Still profiles are something to think about in the future.

**Comment by Kixunil** [comment]
Yeah, definitely! I will do it eventually at least for wallet.

**Comment by jonasnick** [comment]
Should this issue be closed?

**Comment by Kixunil** [comment]
I'm keeping it until I do other examples (at least wallet).
